docs-v2/layouts/partials/info-panel.html

```diff
@@ -45,6 +46,17 @@
             </td>
         </tr>
         {{end}}
+        
+        {{ if $examples}}
+        <tr>
+            <td><b>Examples</b></td>
```

Comment:
nit: can this be "Code Samples" vs examples.  I am not sure if `Examples` is obvious to someone coming to these docs.

Reply:
Makes sense; updated from "Examples" to "Code Samples" (here and in the v1 partial).